modin/logging/logger_function.py

```diff
@@ -0,0 +1,28 @@
+from .config import get_logger
+from functools import wraps
+from modin.config import LogMode
+
+
+def logger_decorator(modin_layer: str, function_name: str, log_level: str):
+    def decorator(f):
+        @wraps(f)
+        def run_and_log(*args, **kwargs):
+            if LogMode.get() != "none":
+                logger = get_logger()
+                try:
+                    getattr(logger, log_level.lower())(
+                        f"START::{modin_layer.upper()}::{function_name}"
+                    )
+                except AttributeError:
+                    raise AttributeError(f"Invalid log_level: {log_level}")
+                result = f(*args, **kwargs)
+                getattr(logger, log_level.lower())(
```

Comment:
This is what I was thinking of in my comment [here](https://github.com/modin-project/modin/pull/4372/files#r862258012).

```suggestion
                try:
                    logger_level = getattr(logger, log_level.lower())
                except AttributeError:
                    raise AttributeError(f"Invalid log_level: {log_level}")                    
                logger_level(f"START::{modin_layer.upper()}::{function_name}")
                result = f(*args, **kwargs)
                logger_level(
```